beakerx/setup.py

```diff
@@ -56,7 +56,15 @@ def update_package_data(distribution):
     # distribution.package_data = find_package_data()
     # re-init build_py options which load package_data
     build_py.finalize_options()
+    magic = distribution.get_command_obj('install_magic')
```

Comment:
i think this should go in https://github.com/twosigma/beakerx/blob/master/beaker/install.py